db/package.js

```diff
@@ -195,6 +200,27 @@ class PackageDbController {
         }
       }))
 
+    // For existing packages, if i exist in the maintainer list, but have a source "invite",
+    // update the source to 'registry'
+    const packageMaintainerUpdates = existingPackages
```

Comment:
let's call this something specific to the source. like maintainer source updates.